spec/helpers/checkout_helper_spec.rb

```diff
@@ -60,6 +60,37 @@
       expect(admin_fee_summary.amount).to eq 123
     end
 
+    context "tax rate adjustments" do
+      let!(:tax_rate) { create(:tax_rate, amount: 0.1, calculator: ::Calculator::DefaultTax.new) }
+      let!(:line_item_fee_adjustment) {
+        create(:adjustment, originator: enterprise_fee, adjustable: order.line_items.first,
+                            order: order)
+      }
+      let!(:order_tax_adjustment) {
+        create(:adjustment,
+          originator: tax_rate,
+          adjustable: fee_adjustment,
+          order: order
+        )
+      }
+      let!(:line_item_tax_adjustment) {
+        create(:adjustment,
+          originator: tax_rate,
+          adjustable: line_item_fee_adjustment,
```

Comment:
On a line item tax adjustment the adjustable would be the line item itself.

Reply:
I think that's just a bad choice of variable name on my part; what I was trying to create was a tax adjustment on the enterprise fee... I think this setup mimics what I was seeing on the invoice from the issue. It had an adjustment with a taxrate originator and an adjustable of the enterprise fee adjustment.